### PR TITLE
 [WIP] new page for new node in kubernetes cluster

### PR DESCRIPTION
**Pull request by Anuragt1104**
Screenshot 2023-12-31 at 12 22 40 AM
Screenshot 2023-12-31 at 12 22 45 AM
Screenshot 2023-12-31 at 12 22 57 AM
Screenshot 2023-12-31 at 12 23 01 AM
Screenshot 2023-12-31 at 12 23 05 AM
 here is the screenshot of the page being added

### CODE REVIEW

**Comment by windsonsea** [comment]
Thanks. You only need to provide a [preview link](https://deploy-preview-44586--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/adding-windows-node/) rather than screenshots across screens.

File: content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+  {{< note >}}
+If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, you have to modify the line: `powershell wins cli process run --path /k/flannel/setup.exe --args "--mode=overlay --interface=Ethernet"` in the `flannel-host-gw.yml` or `flannel-overlay.yml` file and specify your interface accordingly. `bash # Example curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml | sed 's/Ethernet/Ethernet0 2/g' | kubectl apply -f -`
+{{< /note >}}
+
+### Joining a Windows worker node
+
+
+{{< note >}}
+You must install the `Containers` feature and install Docker. Instructions to do so are available at [Install Docker Engine - Enterprise on Windows Servers]().
```
Comment (windsonsea) [inline content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md:110]: Forget to add url in ()?

File: content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+min-kubernetes-server-version: 1.17
+content_type: task
+weight: 110
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
+
+You can use Kubernetes to run a mixture of Linux and Windows nodes, so you can mix Pods that run on Linux on with Pods that run on Windows. This page shows how to register Windows nodes to your cluster.
```
Comment (windsonsea) [inline content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md:12]:
```suggestion
You can use Kubernetes to run a mixture of Linux and Windows nodes, so you can mix Pods
that run on Linux on with Pods that run on Windows. This page shows how to register
Windows nodes to your cluster.
```
Tweaking lone lines around 100 - 120 can provide a clean look for review.

File: content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+
+
+## {{% heading "prerequisites" %}}
+ 
+ {{< version-check >}}
+* Obtain a [Windows Server 2019 license](https://www.microsoft.com/en-us/windows-server/pricing) (or higher) in order to configure the Windows node that hosts Windows containers. If you are using VXLAN/Overlay networking you must have also have [KB4489899](https://support.microsoft.com/en-gb/topic/march-12-2019-kb4489899-os-build-17763-379-2a91a5c5-f351-f181-5501-510308a4030f) installed.
+* A Linux-based Kubernetes kubeadm cluster in which you have access to the control plane (see [Creating a cluster with kubeadm]()).
```
Comment (windsonsea) [inline content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md:19]: Forget to add url in ()?

**Review by windsonsea — COMMENTED** [review]
Seems this is a draft?

File: content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md (hunk 1 above)
Comment (Anuragt1104) [reply content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-node.md]: Thank you for your feedback! I appreciate your attention to detail. I will now add the missing URL in parentheses. Please let me know if there's anything else you'd like me to address.

**Comment by sftim** [comment]
> Thank you for your feedback! I appreciate your attention to detail. I will now add the missing URL in parentheses. Please let me know if there's anything else you'd like me to address.

This PR title is “new page for adding node in kubernetes cluster”, but it only covers Windows nodes. Can you explain the discrepancy?

(Kubernetes also supports Linux nodes)

**Comment by Anuragt1104** [comment]
I'm currently working on the documentation for Linux nodes as well and wanted to streamline the process by submitting a combined pull request. I understand that the current title might not clearly convey this intention.

**Comment by Anuragt1104** [comment]
Okay.

**Comment by Anuragt1104** [comment]
Hi @sftim, I'm currently facing challenges in drafting documentation for Linux nodes. I would greatly appreciate assistance from someone experienced in this domain. Alternatively, if someone else feels more comfortable, I'm open to changing the pull request to focus solely on Windows nodes.

**Comment by sftim** [comment]
I'd prefer not to document only the Windows node story, unless we had a clear plan to also cover setting up Linux nodes.

Most Kubernetes clusters use Linux nodes and ignoring that OS would not be ideal.

**Pull request closed without merging**